### ISSUE TITLE
Unable to create integration

### ISSUE THREAD

**Issue by ZephireNZ** [music-assistant/hass-music-assistant#376]
I am getting the below error when I try to set this up as an integration in Home Assistant.

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 618, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 635, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/mass/__init__.py", line 14, in <module>
    from music_assistant import MusicAssistant
  File "/usr/local/lib/python3.9/site-packages/music_assistant/__init__.py", line 3, in <module>
    from .mass import MusicAssistant  # noqa
  File "/usr/local/lib/python3.9/site-packages/music_assistant/mass.py", line 19, in <module>
    from music_assistant.controllers.music import MusicController
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/__init__.py", line 8, in <module>
    from music_assistant.controllers.music.albums import AlbumsController
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 24, in <module>
    from music_assistant.models.music_provider import MusicProvider
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/music_provider.py", line 7, in <module>
    from music_assistant.models.config import MusicProviderConfig
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/config.py", line 39, in <module>
    class MassConfig:
  File "/usr/local/lib/python3.9/site-packages/music_assistant/models/config.py", line 48, in MassConfig
    stream_port: int = select_stream_port()
  File "/usr/local/lib/python3.9/site-packages/music_assistant/helpers/util.py", line 172, in select_stream_port
    if not is_port_in_use(port):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/helpers/util.py", line 166, in is_port_in_use
    return _sock.connect_ex(("localhost", port)) == 0
socket.gaierror: [Errno -3] Try again
```

<html>
<body>

Version | core-2022.6.6
-- | --
Installation Type | Home Assistant Container
Development | false
Supervisor | false
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.12
Operating System Family | Linux
Operating System Version | 5.18.3-arch1-1
CPU Architecture | x86_64
Timezone | Pacific/Auckland

</body>
</html>

**Comment by marcelveldt** [comment]
OK, looks like port 8095 is already in use and it fails to select a different one. I'll checkin a fix for that.
Temp solution is to disable whatever is now on TCP port 8095.

**Comment by marcelveldt** [comment]
Fix available in 2022.6.1
